packages/design-system/src/components/locale/en.json

```diff
@@ -135,19 +135,19 @@
     "collapse": "Collapse sub-navigation"
   },
   "usaBanner": {
-    "bannerLabel": "Official government website",
+    "flagIconTitle": "U.S. Flag",
```

Comment:
I think this can be deleted. I didn't because it's being called in the flag icon component itself and I don't know if there's a reason that is - it still has `aria-hidden` applied, so doesn't look like it's doing anything.

Reply:
USWDS hides the icon from screen readers?

Reply:
Might be tough to see... but in USWDS's banner, they're using an `<img>` instead of an `<svg>`, so attributes are obviously different, but on the image they have `aria-hidden="true"` and an empty `alt` attr, which tells me this is presentational only.

![Screenshot 2023-07-12 at 10 14 11 AM](https://github.com/CMSgov/design-system/assets/5159392/1fa2763c-5e04-47c3-b04f-153cb11a4d27)

I don't think including a title or aria-label for this icon is actually helpful for AT users because it doesn't provide additional context or information. "U.S. Flag" doesn't really add to an a11y experience imo, so I think USWDS's approach is the correct one.

Reply:
That makes sense. The "An official website of the United States government" seems like a better opening line than "U.S. Flag". While we make it possible for people to use the icon outside of it, in that rare occurance when someone wants to use it, they can add their own title. Removing it cuts the bundle size a bit.